viewflow/views/actions.py

```diff
@@ -37,12 +40,16 @@ def post(self, request, *args, **kwargs):
         self.object = self.get_object()
 
         if self.object.status in [STATUS.DONE, STATUS.CANCELED]:
-            process_message_user(self.request, self.object, "can't be canceled")
+            hyperlink = get_process_hyperlink(self.object)
+            msg = _('Process {hyperlink} can not be canceled.').format(hyperlink)
```

Comment:
should be
```
_('Process {hyperlink} can not be canceled.').format(hyperlink=hyperlink)
```

instead of:
```
_('Process {hyperlink} can not be canceled.').format(hyperlink)
```
The same solution applies to all other uses of named arguments

https://docs.python.org/3/library/string.html#format-examples
https://mkaz.com/2012/10/10/python-string-format/